Add a section for Incident Responders beta (#12271)
DataDog/documentation: content/en/monitors/incident_management/incident_details.md

From the Remediation section, type the description of your task in the creation text box. To assign a task to a Datadog user, type `@` in the description text box, or use the `Assignees` column after the task has been created. An incident task can have more than one assignee. After a task has been created, it can also be assigned a due date. 

As work for different tasks is finished, individual tasks can be marked as completed by clicking the checkbox to the left of the task's description. If you have a large number of tasks, you can filter them down by searching for key words or by hiding completed tasks from view.
  
## Responders section

<div class="alert alert-warning">
This feature is in open beta.
</div>

> {{< img src="monitors/incidents/incident_responders.jpeg" alt="Incident Responders"  style="width:80%;">}}

Reviewer: ```suggestion
{{< img src="monitors/incidents/incident_responders.jpeg" alt="Incident Responders" style="width:80%;">}}
```

> The Responders section is where you form your response team by adding fellow Datadog users and assigning them a role to carry out in the process of resolving the incident. The three default roles provided by Datadog are:

Reviewer: ```suggestion
In the Responders section, you can form your response team by adding other users and assigning them roles to carry out in the process of resolving an incident. The three default roles provided by Datadog are:
```

1. `Incident Commander` - The individual responsible for leading the response team 
2. `Communications Lead` - The individual responsible for managing stakeholder communications throughout the lifecycle of the incident
3. `Responder` - An individual that is actively contribution to investigating an incident and resolving its underlying issue

> **Note:** There must always be exactly one `Incident Commander` at all times during an incident. If there is only one responder in an incident, that individual will be assigned the `Incident Commander` role automatically. There is no limit to the number of individuals that are assigned the `Communications Lead` or `Responder` role in an incident.

Reviewer: ```suggestion
**Note:** There must always be exactly one `Incident Commander` at all times during an incident. If there is only one responder in an incident, this individual is assigned the `Incident Commander` role automatically. There is no limit to the number of individuals that are assigned the `Communications Lead` or `Responder` role in an incident.
```

> If you add another individual as a responder, they will be notified through the email they have associated with their Datadog account. Anyone is able to change the role of a responder, but you can only remove an individual from an incident's Responders List if they have the general `Responder` role and have no activity in the incident. If there is already an `Incident Commander` assigned to an incident, assigning the `Incident Commander` role to another individual will transfer the role over to them. The previous `Incident Commander` will be reassigned the `Responder` role.

Reviewer: ```suggestion
If you add another individual as a responder, they are notified through the email associated with their Datadog account. Anyone is able to change the role of a responder, but you can only remove an individual from an incident's Responders List if they have the general `Responder` role and have no activity in the incident. If there is already an `Incident Commander` assigned to an incident, assigning the `Incident Commander` role to another individual transfers the role over to them. The previous `Incident Commander` is reassigned the `Responder` role.
```
